### ISSUE TITLE
SQLite fails with "no such column: name" when doing a search

### ISSUE THREAD

**Issue by once-ler** [yuki-kimoto/gitprep#96]
When I try to search, I see this error in the production.log

```bash
 [error] DBD::SQLite::db prepare failed: no such column: name. Following SQL is executed.
 select * from "project" where ( name like ?  ) order by name, user_id limit 0, 20
(DBIx::Custom::_create_query) at template /auto/_search.html.ep line 39
74:     # Icky backwards compatibility wrapper. :-(
75:     local @CARP_NOT = $cgc ? $cgc->() : caller();
76:     shortmess_heavy(@_);
```

I've also tried to recreate the gitprep.db like it was shown in README:

```bash
# Upgrade database
cd gitprep
mv data/gitprep.db data/gitprep_v1bak.db
./setup_database
old/copy_database_v1_to_v2 data/gitprep_v1bak.db data/gitprep.db
```

But that did not fix the problem

**Comment by yuki-kimoto** [comment]
@once-ler 

Thanks for reporting bug.

This is bug. "name" is renamed to "id" in GitPrep 2.0, but _search page is not fixed. 

I will fix it.

**Comment by yuki-kimoto** [comment]
I fixed this now.

**Comment by once-ler** [comment]
@yuki-kimoto Thank you for the quick fix.  
It works perfectly from GitHub repo.

However, the tarball does not work 
```
https://github.com/yuki-kimoto/gitprep/archive/latest.tar.gz
```

**Comment by yuki-kimoto** [comment]
I check it tonight. Please wait a little.

**Comment by yuki-kimoto** [comment]
I released v2.2.1 today. latest.tar.gz became new.
